keda/templates/metrics-deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.operatorName }}-metrics-apiserver
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Values.operatorName }}-metrics-apiserver
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Values.operatorName }}-metrics-apiserver
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.operatorName }}-metrics-apiserver
+        {{- if .Values.aadPodIdentity }}
+        aadpodidbinding: {{ .Values.aadPodIdentity }}
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+        imagePullSecrets:
+          {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ .Values.serviceAccount.name }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Values.operatorName }}-metrics-apiserver
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.metricsAdapter }}:{{ .Chart.AppVersion }}"
```

Comment:
As per @ahmelsayed we'd prefer not to do this: https://github.com/kedacore/charts/pull/14#discussion_r372595759

Reply:
sure, let me add the changes for custom keda deployments! :)